src/Administration/Resources/app/administration/src/app/component/extension-api/sw-extension-component-section/sw-extension-component-section.html.twig

```diff
@@ -44,6 +44,13 @@
                 :location-id="componentSection.props.locationId"
             />
         </mt-card>
+        <component :is="componentSection.component" v-else>
```

Comment:
This is nothing we want to support. Because then every component removal would be a breaking change for the SDK. Our goal with the SDK is to have a library which can be used and is stable across several version. So we only want to have defined components for the component renderer.

Example:  
People use `sw-alert` as the component. But this gets removed in 6.8. Therefore it is a break.
A defined component like `card` in general works even when we remove the card. Because we do it in a backward compatible way